src/api/cpp/cvc5.h

```diff
@@ -1266,6 +1259,146 @@ class CVC5_EXPORT Term
    */
   std::wstring getString() const;
 
+  bool isReal32() const;
+  /**
+   * Returns the stored rational as a pair of its numerator and denominator.
```

Comment:
```suggestion
   * Get the representation of a rational value as a pair of its numerator and denominator.
```
I suggest to reword this accordingly in all of the functions below, `stored` is a bit weird (the term must be a value, that's the key point).